genomes/models/genome.py

```diff
@@ -0,0 +1,226 @@
+    )
+
+    img_genome_accession = models.CharField(
+        db_column="img_genome_accession",
+        max_length=20,
+        unique=True,
+        null=True,
+        blank=True,
+    )
+    patric_genome_accession = models.CharField(
+        db_column="patric_genome_accession",
+        max_length=20,
+        unique=True,
+        blank=True,
+        null=True,
+    )
+
+    biome = models.ForeignKey(Biome, db_column="biome_id", on_delete=models.PROTECT)
+
+    length = models.IntegerField(db_column="length")
+    num_contigs = models.IntegerField(db_column="n_contigs")
+    n_50 = models.IntegerField(db_column="n50")
+    gc_content = models.FloatField(db_column="gc_content")
+    type = models.CharField(db_column="type", choices=TYPE_CHOICES, max_length=80)
+    completeness = models.FloatField(db_column="completeness")
+    contamination = models.FloatField(db_column="contamination")
+
+    # EUKS:
+    busco_completeness = models.FloatField(
+        db_column="busco_completeness", null=True, blank=True
+    )
+
+    # EUKS + PROKS:
+    rna_5s = models.FloatField(db_column="rna_5s", null=True, blank=True)
+    # PROKS:
+    rna_16s = models.FloatField(db_column="rna_16s", null=True, blank=True)
+    rna_23s = models.FloatField(db_column="rna_23s", null=True, blank=True)
+    # EUKS:
+    rna_5_8s = models.FloatField(db_column="rna_5_8s", null=True, blank=True)
+    rna_18s = models.FloatField(db_column="rna_18s", null=True, blank=True)
+    rna_28s = models.FloatField(db_column="rna_28s", null=True, blank=True)
+
+    trnas = models.FloatField(db_column="t_rna")
+    nc_rnas = models.IntegerField(db_column="nc_rna")
+    num_proteins = models.IntegerField(db_column="num_proteins")
+    eggnog_coverage = models.FloatField(db_column="eggnog_coverage")
+    ipr_coverage = models.FloatField(db_column="ipr_coverage")
+    taxon_lineage = models.CharField(db_column="taxon_lineage", max_length=400)
+
+    num_genomes_total = models.IntegerField(
+        db_column="pangenome_total_genomes", null=True, blank=True
+    )
+    pangenome_size = models.IntegerField(
+        db_column="pangenome_size", null=True, blank=True
+    )
+    pangenome_core_size = models.IntegerField(
+        db_column="pangenome_core_prop", null=True, blank=True
+    )
+    pangenome_accessory_size = models.IntegerField(
+        db_column="pangenome_accessory_prop", null=True, blank=True
+    )
+
+    annotations = models.JSONField(default=default_annotations, db_column="annotations")
+    default_annotations = staticmethod(default_annotations)
+
+    last_update = models.DateTimeField(db_column="last_update", auto_now=True)
+    first_created = models.DateTimeField(db_column="first_created", auto_now_add=True)
+
+    geo_origin = models.ForeignKey(
+        "GeographicLocation",
+        db_column="geographic_origin",
+        null=True,
+        blank=True,
+        on_delete=models.PROTECT,
+    )
```

Comment:
Thought: why don't we also have these in a JSON field? @MGS-sails @SandyRogers

Reply:
@mberacochea we do. 
But this branch was created prior to that work being done. 
But I've made that change on my genomes api branch which has already been merged